CHANGELOG.md

```diff
@@ -7,9 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
```

Comment:
Simplify these additions to just state what matters to users of the tool.  Remove any Linear references like MAN8-xxxxx.